src/quo2/components/tag.cljs

```diff
@@ -0,0 +1,45 @@
+(ns quo2.components.tag
+  (:require [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo.theme :as theme]
+            [quo2.components.text :as text]))
+
+(def themes {:light {:background-color colors/neutral-20}
+             :dark {:background-color colors/neutral-80}})
+
+(defn get-value-from-size [size big-option small-option]
+  (if  (= size :big) big-option small-option))
+
+(defn tag-container [size border-color]
+  (merge {:height            (if border-color (get-value-from-size size 34 26) (get-value-from-size size 32 24))
+          :align-items        :center
+          :flex-direction     :row
+          :left               0
+          :border-radius      100
+          :padding-right      12}
+         (when border-color {:border-color border-color
+                             :border-width 1})))
+
+(defn tag  "[tag opts \"label\"]
+   opts
+   {
+    :size :small/:big
+    :token-img-src :token-img-src
+    :token-img-style {}
+    :border-color :color
+    :text-value string
+    :overlay child-elements
+    }"
+  [_ _]
```

Comment:
Thanks for adding comments.
From comment it looks like you meant to pass label as argument, but then you used `text-value` ?

Reply:
will adjust!